.github/workflows/console-e2e.yml

```diff
@@ -81,7 +80,7 @@ jobs:
           path: |
             .env/cache/database.pgdump
             .env/admin_api_key.txt
-          key: db-cache-${{ hashFiles('pkg/identityserver/store/**/*.go', 'cmd/ttn-lw-stack/commands/is-db.go', '.github/workflows/e2e.yml', 'docker-compose.yml') }}
+          key: db-cache-${{ hashFiles('pkg/identityserver/**/*.go', 'pkg/identityserver/**/*.sql', 'cmd/ttn-lw-stack/commands/is_db.go', '.github/workflows/console-e2e.yml', 'docker-compose.yml') }}
```

Comment:
Question out of curiosity but why include any go file in the identityserver instead of the files present in `pkg/identityserver/store/migrations/*`. When changes are made to the registries or even the store implementation it shouldn't reflect in a direct change to the DB.

Additionally if there were a script that would be called which invokes operations not present in the files within the migration folder, changes would be that `stack/commands/is_db` is the one calling them.

Reply:
Actually I see why include the changes of the `identityserver/store/**/*.go` files but I still don't see why include the registries on the cache key

Reply:
Because in this case being over specific does not necessarily save that much time - this step takes 45 seconds _in the preparation action_, which is fast enough to warrant some cache misses once in a blue moon.

In contrast, this bug has been here for about a year because the original description is very constrained - it occurred only in PRs which changed the database schema but not `pkg/identityserver/store/**/*.go`. Having a more relaxed hash will indeed have some false-misses but that is fine - should anything change to our package structure this format will still catch it.